scripts/deploy.ts

```diff
@@ -86,8 +87,8 @@ task('deploy:amplforce:testnet', 'Deploy ampleforth contract suite for testnet')
       hre,
       'UFragmentsPolicy',
       deployer,
-      'initialize(address,address,uint256)',
-      [owner, ampl.address, INITIAL_CPI.toString()],
```

Comment:
why don't we have the initial CPI?

Reply:
needs more investigation; initialize() function only provides two vals right now